.github/workflows/release.yaml

```diff
@@ -50,12 +53,6 @@ jobs:
           registry: ghcr.io
           username: ${{ env.GH_USER }}
           password: ${{ secrets.GITHUB_TOKEN }}
-      - name: Login to ECR
```

Comment:
Is there any reason to remove ECR?

Reply:
so sorry about that, I forgot to revert the changes that I've been made for testing purposes.